src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -5,3 +5,70 @@ title: Working with Robots
 description: How to safely handle and work with robots.
 slug: /guides/hardware/working-with-robots
 ---
+
+## Aligning an Ankle
```

Comment:
Not sure what this page is about specifically? If it's aimed at replacing servos maybe the page should be called that. With a section for each servo that you might want to replace